plugins/portlet-mocks/pom.xml

```diff
@@ -51,6 +51,7 @@
     </dependencies>
 
     <properties>
+        <spring.platformVersion>4.3.30.RELEASE</spring.platformVersion>
```

Comment:
good idea, to freeze the spring version here

Reply:
It's the only way to avoid conflict of dependencies :D